cluster/ci/config/submit-queue/submit_queue.yaml

```diff
@@ -254,10 +259,30 @@ objects:
       # pull-logs-dir: pr-logs
       # pull-key: pull
 
+      path-label-config: /path-label/path-label.txt
+
       use-reviewers: "false"
       gate-approved: "true"
       gate-cla: "false"
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: path-label
+  data:
+    path-label.txt: |-
+      # This file is used by the path-label munger and is of the form:
+      # PATH REGEXP     LABEL
+
+      # examples:
+      # pkg/api/types.go
+      # pkg/api/*/types.go
+      ^pkg/api/([^/]+/)?types.go$   needs-api-review
```

Comment:
> Do you want a separate regexp for vendor or are you thinking of matching any "types.go" file in the repo?

Probably `*/[api or apis]/*types.go`, so we don't have predict all the vendor paths.